docs/pages/eas-update/codepush.mdx

```diff
@@ -110,3 +112,22 @@ Please follow the respective store guidelines for submitting a new build of your
 After successfully submitting your app, users will be able to download and use the latest build with EAS Update integration. If your app is not updating as expected, [validate your configuration](/eas-update/debug).
 
 </Step>
+
+## Conceptual differences and incompatibilities between CodePush and EAS Update
+
+CodePush and EAS Update are both services that allow you to send hotfixes to the JavaScript code of your app, but they have some
+
+### Mental model
+
+CodePush has single streams of updates for deployments. How CodePush determines which update to serve is based on rollouts. 
+
+EAS Update has streams of updates that correspond to your source control branches, and additionally, it provides channels that point to those branches. Builds are not directly associated with branches, but rather with channels. Each build points to a single channel, which is set at build time and cannot be modified at runtime. For example (). The reason for this is that it ensures that certain branches (eg: development, staging) don't automatically go out to production -- your preview updates don't go to your production users. This helps you separate the two main uses of EAS Update: previews and production hotfixes.
```

Comment:
```suggestion
EAS Update has streams of updates that correspond to your source control branches, and it provides channels that point to those branches. Builds are not directly associated with branches, but rather with channels. Each build points to a single channel, which is set at build time and cannot be modified at runtime. This is because it ensures that certain branches (such as development and staging) don't automatically go out to production &mdash; your preview updates don't go to your production users. This helps you separate the two main uses of EAS Update: previews and production hotfixes.
```

The example is incomplete.